Common++/src/IpAddress.cpp

```diff
@@ -131,10 +131,8 @@ uint32_t IPv4Address::toInt() const
 
 IPv4Address& IPv4Address::operator=(const IPv4Address& other)
 {
-	if (m_pInAddr != NULL)
-		delete m_pInAddr;
-
-	m_pInAddr = new in_addr();
+	if (m_pInAddr == NULL)
+		m_pInAddr = new in_addr();
```

Comment:
I'm not sure why you did this change. Could you please explain?

Reply:
The same as for IPv6Address.